infra/gcp/terraform/k8s-infra-oci-proxy/main.tf

```diff
@@ -16,8 +16,7 @@ limitations under the License.
 
 // NOTE: compare this file to ./../k8s-infra-oci-proxy-prod/main.tf
 locals {
-  project_id           = "k8s-infra-oci-proxy"
-  service_account_name = "oci-proxy-sandbox"
+  project_id = "k8s-infra-oci-proxy"
```

Comment:
staying a local because it is in the production config. this change reduces the delta between them further.